### DISCUSSION TITLE
👋 Bienvenid@s!

### DISCUSSION THREAD

**Discussion by rramosp** [rramosp/ai4eng.v1#2] [00 - General]
## 👋 Bienvenid@s!

Este espacio de discusiones de la comunidad de **Inteligencia artificial para las ciencias y las ingenierías**. Puedes usarlo para empezar una discusión o participar en cualquiera de las que estén abiertas para, por ejemplo

  * Plantearnos preguntas o dudas sobre el contenido del curso, sus laboratorios, etc.
  * Compartir tus ideas sobre algún aspecto del curso.
  * Encontrar otros estudiantes con quien trabajar conjuntamente.
  * Compartir información relevante, referencias, otros cursos

Hay una categoría de discusiones para **cada módulo del curso**.

Usa la categoría **PLATAFORMA** para cualquier inquietud respecto a la plataforma de autocorrecciones (acceso, problemas, etc.)

Usa la categoría **GENERAL** para cualquier otra cosa.

**Comment by jmaussa** [comment]
Hola, me registre en el curso, pero no me aparece el contenido cuando estoy con mi usuario, solo lo puedo ver en el otro link y no sé entonces como funciona? o como se envían las actividades y que queden con mi nombre?

Solo aparecen el listado de actividades, pero no deja hacer nada

**Reply by rramosp** [reply]
tienes que navegar en el enlace del curso que puse en linkedin ... allí tienes los vídeos, notebooks, talleres, etc.

https://rramosp.github.io/ai4eng.v1

**Reply by ljaimesm** [reply]
Muchas gracias por compartir el conocimiento...
